tiledb/sm/query/readers/sparse_unordered_with_dups_reader.cc

```diff
@@ -378,15 +378,20 @@ bool SparseUnorderedWithDupsReader<BitmapType>::add_result_tile(
     // available memory, allow loading it so we can make progress.
     if (!result_tiles.empty() || tiles_size > available_memory()) {
       logger_->debug(
-          "Budget exceeded adding result tiles, fragment {0}, tile "
-          "{1}",
+          "Total memory budget of {0} exceeded adding result tiles, fragment "
+          "{1}, tile {2}/{3} with size {4}",
+          memory_budget_.total_budget(),
```

Comment:
Can we also print `upper_memory_limit`? It might be the case that the total budget is not exceeded but just the ratio for coordinates

Reply:
Done :+1: